assets/js/theme/account.js

```diff
@@ -234,17 +234,25 @@ export default class Account extends PageManager {
         const validationModel = validation($addressForm, this.context);
         const stateSelector = 'form[data-address-form] [data-field-type="State"]';
         const $stateElement = $(stateSelector);
+        const zipSelector = 'form[data-address-form] [data-field-type="Zip"]';
+        const $zipElement = $(zipSelector);
```

Comment:
Thanks for being consistent here! But if it's Ok for you, let's clean up extra variables here and on line above💅 i think the real reason why they still exist - we haven't touched this for years. And i don't see any benefit of using `stateSelector` or `zipSelector` explicitly.

Reply:
Makes sense, updated